nautobot/core/checks.py

```diff
@@ -1,15 +1,69 @@
 from django.conf import settings
-from django.core.checks import Error, Tags, register
+from django.core.checks import register, Error, Tags, Warning
+from django.core.exceptions import ValidationError
+from django.core.validators import URLValidator
 
 
 E001 = Error(
     "CACHEOPS_DEFAULTS['timeout'] value cannot be 0. To disable caching set CACHEOPS_ENABLED=False.",
-    id="nautobot.E001",
+    id="core.E001",
```

Comment:
Is there any option for fully qualified names (`nautobot.core.E001`)? If `django.core.checks` doesn't like multiple `.` in that way, should we consider `nautobot_core.E001` as a fallback option? To me "core" is not clear enough.

Reply:
I can give it a shot. I was just following suit from Django core with the naming of it by app label.

Reply:
It doesn't appear to care: 

```$ nautobot-server check
SystemCheckError: System check identified some issues:

ERRORS:
?: (nautobot.core.E002) 'nautobot.core.authentication.ObjectPermissionBackend' must be defined in AUTHENTICATION_BACKENDS
?: (nautobot.core.E003) RELEASE_CHECK_TIMEOUT must be at least 3600 seconds (1 hour)
?: (nautobot.core.E004) RELEASE_CHECK_URL must be a valid API URL. Example: https://api.github.com/repos/nautobot/nautobot

WARNINGS:
?: (nautobot.core.W005) STORAGE_CONFIG has been set but STORAGE_BACKEND is not defined. STORAGE_CONFIG will be ignored.

System check identified 4 issues (0 silenced).
```